app/src/components/Button.tsx

```diff
@@ -1,32 +1,59 @@
 import { Trans } from 'next-i18next'
-import { ReactElement } from 'react'
+import React, { MouseEvent, ReactElement } from 'react'
 
-type Props = {
+import { i18nKey } from '@src/types'
+
+export const buttonStyleOptions = [
+  'default',
+  'unstyled',
+  'secondary',
+  'accent-cool',
+  'accent-warm',
+  'base',
+  'outline',
+  'big',
+]
+
+export type ButtonProps = {
   disabled?: boolean
-  labelKey: string
+  labelKey: i18nKey
   style?: string
```

Comment:
Should this be a [union type](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types) equivalent of `buttonStyleOptions`?